src/documentation/pages/components/button/examples/_close-icon-lumen-centurylink.vue

```diff
@@ -0,0 +1,42 @@
+<template lang="pug">
+  <ComponentExample titleSize="h4" title="Close icon button" id="close-icon-lumen-centurylink" :tabs="exampleTabs">
```

Comment:
@soujanreddy what's the difference in Close icon button in the different themes? I think it can be used in the three themes. Please check. Apart of that, titlesize is h3 in chi doc